ee/src/main/java/org/jboss/as/ee/subsystem/ManagedThreadFactoryResourceDefinition.java

```diff
@@ -40,6 +42,12 @@
  */
 public class ManagedThreadFactoryResourceDefinition extends SimpleResourceDefinition {
 
+    /**
+     * the resource definition's dynamic runtime capability
+     */
+    public static final RuntimeCapability<Void> CAPABILITY = RuntimeCapability.Builder.of("org.wildfly.ee.concurrent.threadfactory", true, ManagedThreadFactoryImpl.class)
```

Comment:
For readability, we generally use dashes when converting camel case names to capability names, e.g. "org.wildfly.ee.concurrent.thread-factory"

Reply:
done